src/Input.cpp

```diff
@@ -250,58 +250,60 @@ static void ReadCmakelist(const Configuration& config, std::unordered_map<std::s
     Component &comp = AddComponentDefinition(components, path.parent_path());
     bool inTargetDefinition = false;
     bool inAutoSection = false;
-    int parenLevel = 0;
```

Comment:
This was paren, as in parentheses, not as in parent without a T. This new name is very wrong and confusing.

Reply:
To be honest the parenLevel as in terms of the abbreviation was rather confusing me. Agree that parentLevel is also not great, but I would rather see a different name.

Reply:
When checking again parenLevel is the best typed name will revert back to that.